selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/ShareWorkspaceMemberTest.java

```diff
@@ -89,18 +77,21 @@ public void setUp() throws Exception {
     memberName = testUser.getEmail();
 
     dashboard.open(adminTestUser.getName(), adminTestUser.getPassword());
-    createWorkspace(WORKSPACE_NAME);
   }
 
   @AfterClass
   public void tearDown() throws Exception {
-    workspaceServiceClient.delete(WORKSPACE_NAME, adminTestUser.getName());
+    if (workspaceServiceClient.exists(WORKSPACE_NAME, adminTestUser.getName())) {
+      workspaceServiceClient.stop(WORKSPACE_NAME, adminTestUser.getName());
+      workspaceServiceClient.delete(WORKSPACE_NAME, adminTestUser.getName());
```

Comment:
`workspaceServiceClient.delete(WORKSPACE_NAME, adminTestUser.getName())` is enough so as this method checks on existence of workspace and stops it before deletion.